### PR TITLE
Cleanup Technical Debt 

### PR DESCRIPTION
**Pull request by dimak1**
*Description of changes:*

- Cleanup technical debt
- Rename interface MhrRegistrationHomeOwnersIF -> MhrRegistrationHomeOwnerIF
- Rename composable home-owners -> useHomeOwners

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by cameron-eyds — APPROVED** [review]
Looks good!

File: ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue [hunk 1]
```diff
@@ -271,7 +272,7 @@ export default defineComponent({
   },
   props: {
     editHomeOwner: {
-      type: Object as () => MhrRegistrationHomeOwnersIF,
+      type: Object as () => MhrRegistrationHomeOwnerIF,
```
Comment (ChasBoucher) [inline ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue:275]: Nice... Small Change, but important..

**Review by ChasBoucher — APPROVED** [review]
Looks good...

**Review by doug-lovett — APPROVED** [review]
Very nice.

**Pull request merged**